src/Our.Umbraco.Ditto.Contrib/Collections/ChildrenAs.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+
+namespace Our.Umbraco.Ditto
+{
+    public class ChildrenAs : DittoMultiProcessorAttribute
+    {
+        public ChildrenAs(string documentTypeAlias = null)
```

Comment:
Maybe we should have a constructor that accepts an array of doc types? Not sure if this should then also tie into the DittoDocTypeFactory?